produce.go

```diff
@@ -97,11 +97,19 @@ func (cmd *produceCmd) run(args []string) error {
 	cmd.partitioner = partitioner
 	var err error
 
+	if cmd.avscFile != "" {
+		// Override -valuecodec flag
+		cmd.valueCodecType = "avro"
+		if err := cmd.loadSchemaFile(cmd.avscFile); err != nil {
```

Comment:
Here we should pass the de-referenced schema file right ?

Reply:
yes, we don't handle references :)